packages/tre/src/storage/file.ts

```diff
@@ -317,4 +319,10 @@ export class FileStorage extends LocalStorage {
     }
     return keys;
   }
+
+  getSqliteDatabase(): DatabaseType {
+    if (this.#sqliteDatabase !== undefined) return this.#sqliteDatabase;
+    mkdirSync(path.dirname(this.root), { recursive: true });
+    return (this.#sqliteDatabase = new Database(this.root + ".sqlite3"));
```

Comment:
I could be completely, but this seems like it might put the file somewhere weird?

If I set `this.root` to `~/.mf`, will this not put the database at `~/.mf.sqlite3`? I think I'd expect it to be put inside the root folder